packages/react-conformance/src/defaultTests.tsx

```diff
@@ -65,12 +65,95 @@ export const defaultTests: TestObject = {
+      });
+    }
+  },
+
+  /** Ensures component has top level file in package/src/componentName */
+  'has-top-level-file': (componentInfo: ComponentDoc, testInfo: IsConformantOptions) => {
+    if (!testInfo.isInternal) {
+      it(`has corresponding top-level file 'package/src/Component'`, () => {
+        const { displayName, componentPath, Component } = testInfo;
+        const rootPath = componentPath.replace(/src.*/, '');
+        const topLevelFile = require(path.join(rootPath, 'src', displayName));
+
+        expect(topLevelFile.default || topLevelFile[displayName]).toBe(Component);
+      });
+    }
+  },
+
+  /** If the component is a subcomponent, ensure its parent has the subcomponent as static property */
+  'is-static-property-of-parent': (componentInfo: ComponentDoc, testInfo: IsConformantOptions) => {
+    it(`is a static property of its parent`, () => {
+      const { componentPath, displayName, useDefaultExport } = testInfo;
+      const componentFolder = componentPath.replace(path.basename(componentPath) + path.extname(componentPath), '');
+      const dirName = path.basename(componentFolder).replace(path.extname(componentFolder), '');
+      const isParent = displayName === dirName;
+
+      if (!isParent) {
+        const subComponentName = displayName.replace(dirName, '');
+        const subComponentFile = require(path.join(componentFolder, subComponentName));
+        const SubComponent = useDefaultExport ? subComponentFile.default : subComponentFile[subComponentName];
+
+        expect(ReactIs.isValidElementType(SubComponent)).toBe(true);
+      }
+    });
+  },
+
+  /** Ensures aria attributes are kebab cased */
+  'kebab-aria-attributes': (componentInfo: ComponentDoc, testInfo: IsConformantOptions) => {
+    it(`Aria attributes are kebab-cased`, () => {
+      const props = Object.keys(componentInfo.props);
+
+      for (const prop of Object.keys(props)) {
+        if (prop.startsWith('aria')) {
+          expect(prop).toMatch(/^aria-[a-z]+$/);
+        }
+      }
+    });
+  },
+
+  // TODO: Test last word of callback name against list of valid verbs
+  /** Ensures that components have consistent custom callback names i.e. on[Part][Event] */
+  'consistent-callback-names': (componentInfo: ComponentDoc, testInfo: IsConformantOptions) => {
+    it(`has consistent custom callback names`, () => {
+      const propKeys = Object.keys(componentInfo.props);
+      const { testingOptions } = testInfo;
+      const ignoreProps = testingOptions ? testingOptions['consistent-callback-names']?.ignoreProps : undefined;
+
+      for (const propName of Object.keys(propKeys)) {
+        if (!ignoreProps?.includes(propName) && /^on[A-Z]/.test(propName)) {
+          const words = propName.slice(2).match(/[A-Z][a-z]+/g);
+
+          if (words) {
+            // Make sure last word doesn't end with ed
+            const lastWord = words[words.length - 1];
+            expect(lastWord.lastIndexOf('ed')).not.toBe(lastWord.length - 2);
```

Comment:
You can simplify this since `endsWith` is available in Node:
```suggestion
            expect(lastWord.endsWith('ed')).toBe(false);
```